best-practices/howto_readme.md

```diff
@@ -0,0 +1,68 @@
+# README.md
+
+All projects should have a `README.md` file. This file should try to answer the following questions:
+
+- What it is
+- External requirements
+- How to install it
+- How to use it
+- How to deploy it (if appropriate)
+- How to run the tests
+- Include relevant badges:
+    - [Travis](http://docs.travis-ci.com/user/status-images/)
+    - [Gem Version](http://badge.fury.io/for/rb)
+    - Coverage ([coveralls](https://coveralls.io/))
+    - [Gemnsasium](https://gemnasium.com/)
+    - [waffle](https://waffle.io/)
+
+
+## Example
+
+```
+[![Build Status](https://travis-ci.org/sul-dlss/name-of-the-project.svg?branch=master)](https://travis-ci.org/sul-dlss/name-of-the-project) [![Coverage Status](https://coveralls.io/repos/sul-dlss/name-of-the-project/badge.png)](https://coveralls.io/r/sul-dlss/name-of-the-project) [![Dependency Status](https://gemnasium.com/sul-dlss/name-of-the-project.svg)](https://gemnasium.com/sul-dlss/name-of-the-project) [![Gem Version](https://badge.fury.io/rb/name-of-the-project.svg)](http://badge.fury.io/rb/name-of-the-project)
+
+# Name of the Project
+
+This project provides an implementation of the [XKCD Random Number](https://xkcd.com/221/) generator. This random number generator is particularly useful when predictable random numbers can improve performance.
+
+## Requirements
+
+- Ruby 3.0
+- libxml 5
+
+## Installation
+
+Add this line to your Rails application's Gemfile
+
+\```ruby
```

Comment:
Don't put them in an .md file?  If an example README.md file is required, don't describe it in .md but create it as a template.  Perhaps DLSS best practices can be incorporated into several "template repos", one for each type of project (rails 3, rails 4, sinatra, gem, etc.), e.g.
  - https://github.com/sul-dlss/robot-suite-base